forloop_modules/queries/db_model_templates.py

```diff
@@ -256,8 +256,17 @@ class APIScanWebpage(BaseModel):
 #     dataset_uid: str
 #     sheet_name: str
 #     email: str
-    
-    
+
+
+class APILastActiveDFNodeUid(BaseModel):
+    project_uid: str = "0"
+    last_active_dataframe_node_uid: Optional[str]
+
+
+class APILastActiveDataFrame(BaseModel):
+    project_uid:str = "0"
+    page: int = 1
+    per_page: int = 100
```

Comment:
records_per_page

Reply:
this should not have been here lol
will rename for next pr